app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -82,6 +79,16 @@ class Exposure extends React.Component {
         modified = { patient: { last_date_of_exposure: lde } };
       }
       this.updateLDEandCEValidations({ ...current.patient, [event.target.id]: value });
+    } else if (event?.target?.id && event.target.id === 'no_symptom_history') {
+      // clear out SO if NSH is turned on and populate it with previous SO if NSH is turned off
+      const so = value ? null : this.props.patient.symptom_onset;
```

Comment:
Are you referring to when SO/LDE are previously saved already (saved in database) or not (only previously populated within the enrollment wizard)?

For the first case, this seems to work properly for LDE/CE (see attachment) and for SO/NSH, this technically should never even happen (unless there's a race condition where an assessment is updated from symptomatic to asymptomatic) because if a case has a symptom onset (more specifically, any symptomatic assessments), the user should not be able to check "No Symptom History".

As for the second case, that has never been supported and I think it's fine (unless the PO team suggests otherwise) for this to be the expected behavior.

https://user-images.githubusercontent.com/9956561/112914706-acb0e600-90ca-11eb-8f9f-db776a10b112.mov

Reply:
I see what you're saying. I was only thinking about the second case (previously populated within enrollment wizard). I misunderstood what the comment meant, although after thinking on it more, I think the comment is fine as is.